client/src/components/graph/graph.tsx

```diff
@@ -863,9 +866,15 @@ class Graph extends React.Component<GraphProps, GraphState> {
     asyncProps: GraphAsyncProps,
     prevAsyncProps: GraphAsyncProps | null
   ): void {
-    const { positions, colors, flags, height, width, imageUnderlay } =
-      asyncProps;
-    const { screenCap } = this.props;
+    const {
+      positions,
+      colors,
+      flags,
+      height,
+      width,
+      imageUnderlay,
+      screenCap,
```

Comment:
just wanted to double check, `screenCap` was coming from `this.props`, but now we're getting it from `asyncProps`. Is that intentional? Thanks!

Reply:
it's not necessary to get it from asyncProps here, both places are correct. The reason we're adding screenCap to watchProps is to piggyback on the prop caching.